.github/renovate.json5

```diff
@@ -534,6 +582,12 @@
         ],
         "executionMode": "update"
       }
+    },
+    {
+      "matchDepNames": [
+        "quay.io/cilium/cilium-envoy"
+      ],
+      "versioning": "regex:v(?<major>\\d+)\\.(?<minor>\\d+)\\.(?<patch>\\d+)-.*$",
```

Comment:
thanks, this might require some changes with post upgrade tasks.

I can rebase and adjust the below PR after.

https://github.com/cilium/cilium/pull/33424

Reply:
oh I didn't know you had #33424 open. Yes, let's rebase that one after this one.